packages/dds/tree/src/test/feature-libraries/modular-schema/modularChangeFamily.spec.ts

```diff
@@ -246,64 +227,75 @@ const rootChange1aGeneric: ModularChangeset = {
 };
 
 const rootChange1b: ModularChangeset = {
+	nodeChanges: nestedMapFromFlatList([[nodeId1.revision, nodeId1.localId, nodeChanges1b]]),
 	fieldChanges: new Map([
 		[
 			fieldA,
 			{
-				fieldKind: singleNodeField.identifier,
-				change: brand(nodeChanges1b),
+				fieldKind: genericFieldKind.identifier,
+				change: brand(makeGenericChange(nodeId1)),
 			},
 		],
 	]),
 };
 
 const rootChange1bGeneric: ModularChangeset = {
+	nodeChanges: new Map([[nodeId1.revision, new Map([[nodeId1.localId, nodeChanges1b]])]]),
 	fieldChanges: new Map([
 		[
 			fieldA,
 			{
 				fieldKind: genericFieldKind.identifier,
-				change: brand(
-					genericFieldKind.changeHandler.editor.buildChildChange(0, nodeChanges1b),
-				),
+				change: brand(genericFieldKind.changeHandler.editor.buildChildChange(0, nodeId1)),
 			},
 		],
 	]),
 };
 
+const rebasedChange: ModularChangeset = {
+	nodeChanges: nestedMapFromFlatList([[nodeId1.revision, nodeId1.localId, nodeChanges2]]),
+	fieldChanges: new Map([
+		[
+			fieldA,
+			{ fieldKind: genericFieldKind.identifier, change: brand(makeGenericChange(nodeId1)) },
+		],
+	]),
+};
+
 const rootChange2: ModularChangeset = {
+	nodeChanges: nestedMapFromFlatList([[nodeId2.revision, nodeId2.localId, nodeChanges2]]),
 	fieldChanges: new Map([
 		[
 			fieldA,
 			{
-				fieldKind: singleNodeField.identifier,
-				change: brand(nodeChanges2),
+				fieldKind: genericFieldKind.identifier,
```

Comment:
What was the motivation for changing this from a `singleNodeField` change to `genericFieldKind` change? It makes `rootChange2` and `rootChange2Generic` essential the same and therfore reduces coverage/makes a bunch of tests below redundant.

FWIW, I'd be on board with testing the field normalization logic (which motivated having `rootChange2` and `rootChange2Generic`) separately from the rest of the operations. Happy to chat about it, too.

Reply:
The motivation was that `singleNodeField` didn't really satisfy the field kind contract. I will try to rewrite it and update the tests to use it as part of this PR.